scene/gui/code_edit.h

```diff
@@ -125,6 +125,11 @@ class CodeEdit : public TextEdit {
 
 	/* Line Folding */
 	bool line_folding_enabled = false;
+	String code_region_start_string = "";
+	String code_region_end_string = "";
```

Comment:
```suggestion
	String code_region_start_string;
	String code_region_end_string;
```
These already default to this.

Reply:
I mean not as such, but it's unnecessary, and we usually suggest it elsewhere

Reply:
Nearly all Strings are explicitly set to "" in this file, that's why I chose to keep the same style.
But ok, I'm doing the modification for my new Strings.